### PR TITLE
Feat(respec2html): upgrade to command-line-usage V3

### PR DESCRIPTION
**Pull request by marcoscaceres**
V3 of command-line-args was a breaking change. Update fixes issues and makes a few enhancements.

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/respec/815)

screenshot 2016-06-08 18 11 17

### CODE REVIEW

**Comment by marcoscaceres** [comment]
Added picture of new output ... so pretty 😻

**Comment by marcoscaceres** [comment]
@dontcallmedom, you are the primary consumer of this feature. Can you ptal? The changes are all cosmetic, but always appreciate your input.

File: tools/respec2html.js [hunk 1]
```diff
@@ -40,47 +44,59 @@ const optionDefinitions = [{
...
   let parsedArgs;
   try {
-    parsedArgs = cli.parse();
+    parsedArgs = commandLineArgs(optionList);
   } catch (err) {
-    console.error(err.stack);
-    tasks.showHelp();
-    process.exit(2);
+    console.info(getUsage(usageSections));
+    console.error(colors.red(err.message));
```
Comment (dontcallmedom) [inline tools/respec2html.js]: given what I see on line 111, shouldn't this be` colors.error`

Comment (marcoscaceres) [inline tools/respec2html.js]: good catch.

**Pull request merged**